frontend/services/entities/hosts.ts

```diff
@@ -242,27 +218,31 @@ export default {
       },
     });
   },
-  exportHosts: (options: IExportHostsOptions) => {
```

Comment:
This is the fix.

Reply:
`osName`, `osVersionId`, `osVersion`, `bootstrapPackageStatus` did not exist (released bugs)